### ISSUE TITLE
time: TestLoadFixed fails on 1.7.3 source build

### ISSUE THREAD

**Issue by karalabe** [golang/go#18064]
I've just set up a new machine and wanted to build Go and was surprised to see that 1.7.3 compilation always fail on a time test. My build procedure is as follows:

* (My OS is Ubuntu Yakkety)
* Download Go 1.7.3 for Linux amd64 and set it up as `GOROOT-BOOTSTRAP`
* Clone the upstream Go repository
* Checkout `go1.7.3` (latest commit at `2f6557233c5a5c311547144c34b4045640ff9f71`)
* Run `all.bash` from the `src` folder

This results in:

```
--- FAIL: TestLoadFixed (0.00s)
	time_test.go:943: Now().In(loc).Zone() = "-01", -3600, want "GMT+1", -3600
FAIL
FAIL	time	2.439s
```

**Comment by bradfitz** [comment]
Dup of #17276, fixed by c5434f2973a87acff76bac359236e690d632ce95

You can ignore that test failure. It's caused by an updated tzdata that came out after that version of Go was released. It'll be fixed in Go 1.8.

**Comment by karalabe** [comment]
Ah ok, thanks! :)
